app/uk/gov/hmrc/eoricommoncomponent/frontend/views/migration/nameId.scala.html

```diff
@@ -32,7 +32,7 @@
 )(implicit request: Request[_], messages: Messages)
 
 @headerAndTitle = @{
-    if(displayMode=="partnership") messages("cds.matching.header.partnership")
+    if(displayMode=="partnership" || displayMode=="Llp-partnership") messages("cds.matching.header.partnership")
```

Comment:
Seen here and elsewhere in the file are these magic string checks. Lets try and add some type safety here. Lets make DisplayMode into a trait and a set of case objects that can inform us here.